esp/useful_scripts/server_setup/new_site.sh

```diff
@@ -1,20 +1,35 @@
 #!/bin/bash
 
 # ESP site creation script
-# Michael Price, December 2010
+# See /docs/dev/deploying.rst for documentation on how to deploy.
+
+set -e -o pipefail
```

Comment:
Ah -- the modes are all undefined if not in use, and it would be kind of annoying to define all of those.  So I'll skip `-u`.